config-connector/solutions/projects/kpt/project-services/README.md

```diff
@@ -0,0 +1,42 @@
+Project Services
+==================================================
+
+# NAME
+
+  project-services
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a project.
+
+## Requirements
+  Your service account must have roles/serviceusage.serviceUsageAdmin or roles/owner
```

Comment:
Maybe quoting the roles using backticks to highlight - `roles/serviceusage.serviceUsageAdmin` or `roles/owner`?

And I think we'll want to be more explicit about the service account mentioned here.

Reply:
I've highlighted the roles. I'm not sure if the service account needs to be explicitly specified since it is part of Config Connector.